test-server.js

```diff
@@ -27,12 +30,25 @@ describe('test server', function () {
             email: 'jimmy@gmail.com',
             password: 'iamnotfondoficecream1234',
         };
-        request(server).get('/api/get-user/2').expect(200).end((err, res) => {
-            expect(res.body.user.firstName).to.equal(user.firstName);
-            expect(res.body.user.email).to.equal(user.email);
-            expect(res.body.user.password).to.equal(user.password);
-            done();
-        });
+        request(server)
+            .get('/api/get-user/2')
+            .expect(200)
+            .end((err, res) => {
+                expect(res.body.user.firstName).to.equal(user.firstName);
```

Comment:
The `err` parameter here was not used. In this simple use case that would probably be fine, however, it's good practice to just handle this properly with an error check. Adding the check would ensure that Mocha knows the test has finished in the event of an error. This check would help prevent hanging tests or false positives

```suggestion
                if (err) return done(err);
                expect(res.body.user.firstName).to.equal(user.firstName);
```